src/Cosserat/constraint/CosseratActuatorConstraint.cpp

```diff
@@ -47,8 +48,11 @@ using namespace sofa::core;
 // 1-RegisterObject("description") + .add<> : Register the component
 // 2-.add<>(true) : Set default template
 
-int CosseratActuatorConstraintClass = RegisterObject("Simulate cable actuation.")
-.add< CosseratActuatorConstraint<Vec3Types> >(true)
+void registerCosseratActuatorConstraint(sofa::core::ObjectFactory* factory)
```

Comment:
The comments before must be removed as it is no longer consistent with the new mechanism. This has to be done everywhere

Reply:
Hi @alxbilger. 

It's done in this [PR](https://github.com/bakpaul/Cosserat/pull/1), @bakpaul, please can you merge it ?

Reply:
no it was not done. I've done it in [this commit](https://github.com/SofaDefrost/Cosserat/pull/141/commits/373aa6957139fa0bbf00501e12b85219bbe9c4ee)